ch.sbb.scion.rcp.microfrontend.e3.app.demo/src/ch/sbb/scion/rcp/microfrontend/e3/app/demo/view/model/ApplicationQualificationCheckModel.java

```diff
@@ -0,0 +1,32 @@
+package ch.sbb.scion.rcp.microfrontend.e3.app.demo.view.model;
+
+import org.eclipse.core.databinding.observable.value.IObservableValue;
+import org.eclipse.core.databinding.observable.value.WritableValue;
+
+public class ApplicationQualificationCheckModel {
+
+  private final IObservableValue<String> capabilityId = new WritableValue<>("", String.class);
+
+  private final IObservableValue<String> appSymbolicName = new WritableValue<>("", String.class);
+
+  public IObservableValue<String> getCapabilityId() {
+    return capabilityId;
+  }
+
+  public IObservableValue<String> getAppSymbolicName() {
+    return appSymbolicName;
+  }
+
+  public void clearValues() {
+    capabilityId.setValue("");
+    appSymbolicName.setValue("");
+  }
+
+  public AppSymbolicNameAndCapabilityId getAppSymbolicNameAndCapabilityId() {
+    return new AppSymbolicNameAndCapabilityId(appSymbolicName.getValue(), capabilityId.getValue());
+  }
+
+  public record AppSymbolicNameAndCapabilityId(String appSymbolicName, String capabilityId) {
+
```

Comment:
intentionally empty?

Reply:
Yes - that's just a value holder, so that I can return two values from the getter, and it also ties the two values together. (And it is not really empty, a record has getters and a required args ctor 🙂)